libraries/rush-lib/src/logic/base/BaseInstallManagerTypes.ts

```diff
@@ -84,8 +84,18 @@ export interface IInstallManagerOptions {
   /**
    * Filters to be passed to PNPM during installation, if applicable.
    * These restrict the scope of a workspace installation.
```

Comment:
This description is too vague.

```
* An array of `--filter` arguments.  For example, if the array is ["a", "b"] then Rush would invoke
* will use `pnpm install --filter a --filter b` which restricts the install/update to dependencies of
* workspace projects "a" and "b".  If the array is empty or undefined, then an unfiltered install
* is performed.
```

Reply:
This is actually just an array of arguments, including the `--filter` arguments.